micro-ui/web/micro-ui-internals/packages/modules/workbench/src/hooks/inbox.js

```
@@ -0,0 +1,116 @@
+    [url, params, body, "popup"].filter((e) => e),
+    fetchData,
+    {
+      cacheTime: 0,
+      ...config,
+      select: (data) => {
+        const respData = data?.mdms?.map((e) => ({ label: e?.uniqueIdentifier, value: e?.uniqueIdentifier }));
+
+        if (schema?.definition?.["x-ref-schema"]?.length > 0) {
+          schema?.definition?.["x-ref-schema"]?.map((dependent) => {
+            if (dependent?.fieldPath) {
+              let updatedPath = Digit.Utils.workbench.getUpdatedPath(dependent?.fieldPath);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updatedPath' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)